src/client/api-rpc.test.js

```diff
@@ -1,11 +1,11 @@
-import { cryptoWaitReady } from "@polkadot/util-crypto";
-import { ApiRpc } from "./api-rpc";
-import { DockRpc } from "./dock-rpc";
+import {cryptoWaitReady} from '@polkadot/util-crypto';
+import {ApiRpc} from './api-rpc';
+import {DockRpc} from './dock-rpc';
 
-describe("ApiRpc", () => {
+describe('ApiRpc', () => {
   beforeAll(cryptoWaitReady);
 
-  it("getAccountBalance", async () => {
+  it('getAccountBalance', async () => {
     const result = await ApiRpc.getAccountBalance();
```

Comment:
This is testing the layer that uses RPC to hit the polkadot endpoints right? Does it actually reach out to the chain as part of the test?

Reply:
No, this test has changed. There is a mock being used.